index.ts

```diff
@@ -81,6 +81,11 @@ function initApp(): void {
   rightsManagement.setRequiredRights(
     getRequiredRights(workspaceOwnerRightsNeeded, inProgressOwnerRightsNeeded)
   );
+  const cookieSettings: CookieOptions = {
+    maxAge: 60 * 60 * 1000, // 1 hour,
+    secure: authenticationMethod === 'SSL',
+    sameSite: authenticationMethod === 'SSL' ? 'strict' : 'lax'
```

Comment:
don't do stuff like this without testing; rather have the ssl method break and we figure it out later